.gitignore

```diff
@@ -150,6 +150,7 @@ publish/
 *.nuget.targets
 *.nupkg
 **/packages/*
+**/localpackages/*
```

Comment:
I would prefer we keep this under the bin folder as it is a product of the build. Perhaps "bin/packages/source" or something like that.